reactComponents/src/searchbar/tokenSearch/ResultDetail.tsx

```diff
@@ -273,12 +274,12 @@ export const ResultDetail: FC<DetailType> = (props: DetailType) => {
           </div>
           <div className="actions">
             {customActions &&
-              customActions.map((action) => (
+              customActions.map((action, index) => (
                 <Action
-                  key={`action-${action.index}`}
-                  component={action.component}
+                  key={`action-${index}`}
```

Comment:
what is this change for? I don't see it in the MR comments

Reply:
In rome-frontend, we passed custom actions like this.
const customActions = [
    {
      component: chartAction,
      index: 1,
    },
    {
      component: tableAction,
      index: 2,
    },
    {
      component: exchangeAction,
      index: 3,
    },
 ];

But I don't think we need `index` here. So I removed `index` property and passed it as Array of component. 

Note:
In terminal library, I don't remember that we use PR templates like rome-frontend.